tmp.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Is this included by accident?  If not it could really use a better file name than "tmp.json" unless the workflow demands it.

Reply:
Whoops, had that as a testing file and forgot to remove it.